src/main/java/com/to/ordermanagementservice/BasketTest.java

```diff
@@ -0,0 +1,36 @@
+package com.to.ordermanagementservice;
+
+import com.to.ordermanagementservice.entity.*;
```

Comment:
Configure your IDE not to set wildcard imports. It is possible to set min number of imports to start using wildcards to  99